src/Uno.UWP/Devices/Sensors/HingeAngleSensor.Android.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Threading.Tasks;
+using Uno.Devices.Sensors;
+using Uno.Foundation.Extensibility;
+using Windows.Foundation;
+using Uno.Logging;
+using Uno.Extensions;
+using Uno.Devices.Sensors.Helpers;
+
+namespace Windows.Devices.Sensors
+{
+	public partial class HingeAngleSensor
+	{
+		private readonly static object _syncLock = new object();
+
+		private static bool _initializationAttempted = false;
+		private static HingeAngleSensor _instance;
+		private static INativeHingeAngleSensor _hingeAngleSensor = null;
+
+		private TypedEventHandler<HingeAngleSensor, HingeAngleSensorReadingChangedEventArgs> _readingChanged;
+
+		private HingeAngleSensor()
+		{
+		}
+
+		public static IAsyncOperation<HingeAngleSensor> GetDefaultAsync()
```

Comment:
Does GetDefaultAsync return a dummy HingeAngleSensor if there is none ?

Reply:
If there is no hinge on the device, the GetDefaultAsync method returns null, which is in line with other sensors when not available on the device. Also in case the SDK is not referenced, there is logging to inform the developer.